nasa-gibs/worldview
# WV-2685: Opera Water Layer Notice

## Description

Disabling snapshots for new OPERA water layer.

## How To Test

1. `git checkout wv-2685-opera-water-layer-notice`
2. `npm run build`
3. `npm run watch`
4. Add OPERA water layer.
5. Try to take a snapshot with the water layer visible.
6. Verify notification message.
